RiotSwiftUI/Modules/UserSessions/UserSessionOverview/View/UserSessionOverviewItem.swift

```diff
@@ -40,17 +45,27 @@ struct UserSessionOverviewDisclosureCell: View {
             .background(theme.colors.background)
         }
     }
+    
+    var textColor: Color {
+        isDestructive ? theme.colors.alert : theme.colors.primaryContent
+    }
 }
 
-struct UserSessionOverviewDisclosureCell_Previews: PreviewProvider {
+struct UserSessionOverviewButtonCell_Previews: PreviewProvider {
```

Comment:
Not that it makes any difference but this should be `UserSessionOverviewItem_Previews`